python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -46,12 +49,13 @@ def run_subcommand(argv: list[str]) -> Result[str, str]:
         LOGGER.debug("Running edit subcommand")
         res_edit_config = core_utils.parse_args(main_parser, argv[1:], EditServerConfig)
         LOGGER.debug(f"{res_edit_config.is_ok()=}")
-        res_servers = res_edit_config.and_then(_run_editor_servers)
         out: Result[str, str] = result.do(
             #
             Ok(",".join(res_servers_ok))
             #
-            for res_servers_ok in res_servers
+            for edit_config in res_edit_config
+            for cli_config in res_cli_config
+            for res_servers_ok in _run_editor_servers(edit_config, cli_config.config_path)
```

Comment:
trying to understand this code block, what is going on here? Do `edit_config` and `cli_config` get used?

Reply:
Yes they get passed into _run_editor_servers. Specifically cli_config.config_path.